python/PEDvPUDplot.py

```diff
@@ -2,104 +2,166 @@
 """
 Created on Sat Nov  7 12:59:47 2015
 
-@author: ymseah
+Make scatterplot of pairwise uncorrected vs evolutionary distances.
+
+Pairwise uncorrected distances are obtained from SDT 1.0 (filename_mat.txt), while pairwise maximum likelihood distances are obtained from Tree-Puzzle 5.2 (outdist.txt). Both files are read to extract sequence ID and distance values.
+
 """
 
 import numpy as np
 import matplotlib.pyplot as plt
-import itertools # for zipping, less memory consumption than zip
+import itertools
 
 filepath = raw_input("Enter path to SDT matrix file: ")
```

Comment:
I would highly recommend moving these input variables down to where you actually run the script.

Change the names too for clarity:
filepath => sdtMatrixFilePath
filepath2 => treePuzzleDistanceFilePath